src/repl/scala/tools/nsc/interpreter/Phased.scala

```diff
@@ -78,13 +78,13 @@ trait Phased {
     val saved = get
     set(ph)
     try atCurrent(body)
-    finally set(saved)
+    finally { set(saved); () }
```

Comment:
I had quite a few nowarns on finally exprs. Often booleans indicating success/failure.

I think an annotation is a marker for "this is side-effecting" or "this is throwaway code, let it fail silently" (or maybe test code and I'm sloppy).